File: dataset/human/java/file_7900.java
import  java.util.*;
class Main{
    int s;
    int i;
    int a=0;
    Main(){
	Scanner sc=new Scanner(System.in);
	for(i=1; i<11; i++){
	   s=sc.nextInt();
	   a=a+s;
	}
	System.out.println(a);	    
	  
	

    }

    public static void main(String[] args){
	new Main();
    }
}

File: dataset/human/python/file_7900.py
import scala.io.StdIn._
object Main extends App{
  val sc = new java.util.Scanner(System.in)
  var s = sc.next
  if(s.contains("a")&&s.contains("b")&&s.contains("c")){
    println("Yes")
  }
  else{
    println("No")
  }
}
